subsys/bluetooth/controller/ll_sw/nordic/lll/lll_sync.c

```diff
@@ -36,24 +36,38 @@
 
 #include "ll_feat.h"
 
+#include "ull_internal.h"
+#include "ull_sync_types.h"
```

Comment:
Should not use ULL in LLL, if there is any member that  needs access by LLL, should be placed in LLL types or extern interface be added to get it.

Reply:
Moved `enum sync_status` to lll_sync.h and removed ULL includes from the file.